logico/Main.pl

```diff
@@ -1,211 +1,231 @@
+opcao_escolhida_admin(1) :- 
+    writeln("Cadastro Admin"),
+    writeln("Insira login do novo admin:"),
+    read(Login),
+    writeln("Insira senha do novo admin:"),
+    read(Senha),
+    tty_clear,
+    cadastro_admin(Login, Senha, R),
+    writeln(R), 
+    opcoes_menu_admin,
+    read(Opcao),
+    opcao_escolhida_admin(Opcao).
+
+opcao_escolhida_admin(2) :- 
+    writeln("Remove administrador"),
+    writeln("Insira login do admin a ser removido:"),
+    read(Login),
+    tty_clear,
+    remove_admin(Login, R),
+    writeln(R), 
+    opcoes_menu_admin,
+    read(Opcao),
+    opcao_escolhida_admin(Opcao).
+
+opcao_escolhida_admin(3) :- 
+    writeln("Edita senha do administrador"),
+    writeln("Insira login do admin a ser editado:"),
+    read(Login),
+    writeln("Insira sua nova senha:"),
+    read(NovaSenha),
+    tty_clear,
+    edita_admin(Login, NovaSenha, R),
+    writeln(R), 
+    opcoes_menu_admin,
+    read(Opcao),
+    opcao_escolhida_admin(Opcao).
+
+opcao_escolhida_admin(4) :- 
+    writeln("Cadastro Estudantes"),
+    writeln("Insira o caminho (entre aspas simples ou duplas) para o arquivo .csv que deve conter duas colunas (matricula e senha) para cada estudante"),
+    read(Caminho),
+    tty_clear,
+    cadastro_estudantes(Caminho, R),
+    writeln(R), 
+    opcoes_menu_admin,
+    read(Opcao),
+    opcao_escolhida_admin(Opcao).
+
+opcao_escolhida_admin(5) :- 
+    writeln("Desativar Estudante"),
+    writeln("Insira a matrícula do estudante que será desativado"),
+    read(Matricula),
+    tty_clear,
+    desativar_estudante(Matricula, R),
+    writeln(R),
+    opcoes_menu_admin,
+    read(Opcao),
+    opcao_escolhida_admin(Opcao).
+
+opcao_escolhida_admin(6) :-
+    writeln("Cadastro de votação"),
+    writeln("Insira a data da nova votação:"),
+    read(DataVotacao),
+    cadastro_votacao(DataVotacao, R),
+    tty_clear,
+    writeln(R),
+    opcao_menu_cadastro_votacao(),
+    read(Opcao),
+    opcao_escolhida_votacao(Opcao).
+
+opcao_escolhida_admin(8) :-
+    writeln("Encerrar votação"),
+    writeln("Insira o id da votação que deseja encerrar:"),
+    read(IdVotacao),
+    encerra_votacao(IdVotacao, R),
+    tty_clear,
+    writeln(R),
+    opcoes_menu_admin,
+    read(Opcao),
+    opcao_escolhida_admin(Opcao).
+
+opcao_escolhida_admin(9) :- 
+    tty_clear,
+    opcoes_menu_principal,
+    read(Opcao),
+    opcao_escolhida_principal(Opcao).
+
+opcao_escolhida_estudante(1, Matricula) :- 
+    writeln("Edita senha do estudante"),
+    writeln("Insira nova senha:"),
+    read(NovaSenha),
+    tty_clear,
+    editar_senha_estudante(Matricula, NovaSenha, R),
+    writeln(R), 
+    opcoes_menu_estudante,
+    read(Opcao),
+    opcao_escolhida_estudante(Opcao, Matricula).
+
+opcao_escolhida_estudante(3, _) :- 
+    tty_clear,
+    opcoes_menu_principal,
+    read(Opcao),
+    opcao_escolhida_principal(Opcao).
+
+opcao_escolhida_votacao(1) :- 
+    writeln("Cadastro Chapa"),
+    writeln("Insira o nome da Chapa"),
```

Comment:
Aqui cabe um aviso de usar aspas porque 'nova chapa' é cadastrada se for usada com '' mas sem elas, dá erro.

Reply:
não entendi :/

Reply:
Muda o enunciado pra algo do tipo "Insira o nome da Chapa (entre aspas simples ou duplas)"